code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1066,7 +1066,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		chem.overdosed = TRUE
 
 /datum/species/proc/check_species_weakness(obj/item, mob/living/attacker)
-	return 0 //This is not a boolean, it's the multiplier for the damage that the user takes from the item.It is added onto the check_weakness value of the mob, and then the force of the item is multiplied by this value
+	return 1 //This is not a boolean, it's the multiplier for the damage that the user takes from the item.It is added onto the check_weakness value of the mob, and then the force of the item is multiplied by this value
```

Comment:
this is possibly where melee is being broken?

Reply:
Melee was fixed from before this review